src/System.Private.CoreLib/shared/System/Resources/ManifestBasedResourceGroveler.cs

```diff
@@ -384,11 +384,13 @@ private Assembly GetSatelliteAssembly(CultureInfo lookForCulture)
                 // Ignore cases where the loader gets an access
                 // denied back from the OS.  This showed up for
                 // href-run exe's at one point.  
+#if CORECLR
                 int hr = fle.HResult;
                 if (hr != Win32Marshal.MakeHRFromErrorCode(Interop.Errors.ERROR_ACCESS_DENIED))
                 {
                     Debug.Fail("[This assert catches satellite assembly build/deployment problems - report this message to your build lab & loc engineer]" + Environment.NewLine + "GetSatelliteAssembly failed for culture " + lookForCulture.Name + " and version " + (_mediator.SatelliteContractVersion == null ? _mediator.MainAssembly.GetName().Version.ToString() : _mediator.SatelliteContractVersion.ToString()) + " of assembly " + _mediator.MainAssembly.GetName().Name + " with error code 0x" + hr.ToString("X", CultureInfo.InvariantCulture) + Environment.NewLine + "Exception: " + fle);
                 }
+#endif
```

Comment:
Not sure this is worth keeping at all (or testing for ERROR_ACCESS_DENIED specifically).

Reply:
I think it would be fine to delete it.